assets/css/general.css

```diff
@@ -5,5 +5,5 @@
 }
 
 body {
```

Comment:
move this to the structure file. then you can remove this file.

Reply:
Done.